### DISCUSSION TITLE
Ntly 245 tested on iOS

### DISCUSSION THREAD

**Discussion by svenduvoisin** [SRGSSR/pillarbox-apple#298] [Feedbacks]
- when rotating, the white background is visible.
- while seeking with the scrubber, if I stop moving but keep the finger on the scrubber, it plays strangely.
- Live streams (RSI) always slow to start.
- Live button behavior (RSI) after seeking and back to live.
- when rotating (4K), the player window closes.
- [Stories] window layout of the top screen ?
- [Stories] no Play/Pause button, no restart from beginning after swiping.
- [Playlist] left arrow visible even if first element is playing - should it be a start over button ?
- playback speed not yet implemented ?

**Comment by defagos** [comment]
Thank you very much for the feedback.

> when rotating, the white background is visible.

Added to drafts. Not sure how we can fix this one, though (I already tried without much success).

> while seeking with the scrubber, if I stop moving but keep the finger on the scrubber, it plays strangely.

Expected behavior since the user is forcing the player to stay at a given location, while the player wants to move forward. This means that the player will keep buffering again and again.

> Live streams (RSI) always slow to start.

Should be monitored but mostly depends on metadata and stream delivery.

> when rotating (4K), the player window closes.

Known issue already in our backlog.

> [Stories] window layout of the top screen ?

Modal presentation is partial to avoid having to implement aa close button, but full screen would be feasible of course.

>  [Stories] no Play/Pause button, no restart from beginning after swiping.

Intended behavior. Such experiences usually do not provide any playback button. We also reuse neighboring players to show that such experiences can be achieved, this is why the position is kept when swiping between neighboring stories. So for the moment this is intended behavior.

> [Stories] no Play/Pause button, no restart from beginning after swiping.

Depends on the _Smart navigation_ setting but intended behavior. Apps could easily implement a startover button if desired as well, but we kept our playlist demo layout tight.

> playback speed not yet implemented ?

Nope but will be at a later time.
